test/controllers/posts_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class PostsControllerTest < ActionDispatch::IntegrationTest
```

Comment:
please google how to turn off tests generators :). We'll be using rspec anyway

Reply:
Additional generated files removed